packages/ui-components/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @cloudoperators/juno-ui-components
 
+## 2.36.1
+
+### Patch Changes
+
+- e8492fd: test dependencies upgrade with changeset
```

Comment:
Do we want to remove this changeset. This was just a test we did?

Reply:
So, you mean to not to include it in Changelog? Sure!

Reply:
done